samples/core/src/test/java/io/oasp/gastronomy/restaurant/salesmanagement/service/impl/rest/SalesmanagementHttpRestServiceTest.java

```diff
@@ -61,6 +63,9 @@
   private SalesmanagementRestServiceTestHelper helper;
 
   @Inject
+  private ServiceClientFactory serviceClientFactory;
```

Comment:
Could the injection use Generics?
```
@Inject
private ServiceClientFactory<SalesmanagementRestService> serviceClientFactory;
```

So later we don't need to specify the class?
`this.service = this.serviceClientFactory.create()`